spec/aggregate_root_spec.rb

```diff
@@ -3,16 +3,18 @@
 class Order
   include RailsEventStore::AggregateRoot
 
+  attr_accessor :id, :status
```

Comment:
I've pushed 2 new commits - the tests are still not perfect - I will improve them after I finish my blog post.